platform_main_feature.views_default.inc

```diff
@@ -1383,14 +1383,13 @@ function platform_main_feature_views_default_views() {
   $handler->display->display_options['pager']['options']['offset'] = '0';
   $handler->display->display_options['style_plugin'] = 'default';
   $handler->display->display_options['row_plugin'] = 'fields';
-  /* Field: Islandora Solr: mods_titleInfo_subTitle_ms */
+  /* Field: Broken/missing handler */
   $handler->display->display_options['fields']['mods_titleInfo_subTitle_ms']['id'] = 'mods_titleInfo_subTitle_ms';
   $handler->display->display_options['fields']['mods_titleInfo_subTitle_ms']['table'] = 'islandora_solr';
   $handler->display->display_options['fields']['mods_titleInfo_subTitle_ms']['field'] = 'mods_titleInfo_subTitle_ms';
   $handler->display->display_options['fields']['mods_titleInfo_subTitle_ms']['label'] = '';
   $handler->display->display_options['fields']['mods_titleInfo_subTitle_ms']['alter']['strip_tags'] = TRUE;
   $handler->display->display_options['fields']['mods_titleInfo_subTitle_ms']['element_label_colon'] = FALSE;
-  $handler->display->display_options['fields']['mods_titleInfo_subTitle_ms']['link_to_object'] = 0;
```

Comment:
Not really an issue. A little broken, 'til the cache gets cleared, but I think that's more to do with the 'Broken/missing handler' business.